server/app.spec.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable max-lines */
```

Comment:
This test probably should be located alongside the other api tests:

https://github.com/naturalcrit/homebrewery/blob/master/server/homebrew.api.spec.js

You may find a simpler way to do this following the tests we added for "getThemeBundle()"

Reply:
I shifted the function and the tests to the homebrew.api files.